src/pages/index.tsx

```diff
@@ -12,23 +12,27 @@ import Head from 'next/head';
 import React, { useMemo } from 'react';
 import { Sponsors } from '@/components/Sponsors';
 import { LeaveFeedback } from '@/components/LeaveFeedback';
-import EventsList from '@/components/EventsList';
+import EventsList from '@/components/event/EventsList';
 import { Newsletter } from '@/components/Newsletter';
 
 type Props = {
   events: [IEvent];
 };
 
+const MAX_PAST_EVENTS = 3;
+
 const Home: NextPage<Props> = ({ events: events }: Props) => {
   const pastEvents = useMemo(
-    () => sortEvents(filterPastEvents(events)),
+    () => sortEvents(filterPastEvents(events)).slice(0, MAX_PAST_EVENTS),
     [events]
   );
   const nextEvents = useMemo(
     () => sortEvents(filterComingEvents(events)),
     [events]
   );
 
+  const hasMorePastEvents = events.length > MAX_PAST_EVENTS + nextEvents.length;
```

Comment:
Maybe I didn't get the logic but I can't understand why `nextEvents` should be involved in this condition
```suggestion
  const hasMorePastEvents = filterPastEvents(events).length > MAX_PAST_EVENTS;
```
Am I losing something?

Reply:
Indeed, my bad 👍